tests/test-sessions/test-gcloud-sessions/pom.xml

```diff
@@ -78,6 +94,7 @@
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
               <skipTests>false</skipTests>
+              <trimStackTrace>false</trimStackTrace>
               <systemPropertyVariables>
                 <DATASTORE_DATASET>jetty9-work</DATASTORE_DATASET>
                 <DATASTORE_HOST>http://localhost:8088</DATASTORE_HOST>
```

Comment:
I don't think lines 99-100 are needed anymore are they?

Reply:
ah yup. We will need that in the top pom :)